models/intermediate/int_apple_search_ads__most_recent_creative_set.sql

```diff
@@ -1,14 +1,15 @@
+--
```

Comment:
When looking at the DAG, this model doesn't seem to be leveraged downstream. Is the intention that this will be used in the Ad Reporting package? Or is it actually not needed?

![image](https://user-images.githubusercontent.com/74217849/150832447-0fd6f959-eab7-4406-b3e9-587d350a9df1.png)

Reply:
Or is it supposed to flow into the `apple_search_ads__creative_set_report` model?

Reply:
ah good catch joe. I think I originally added it because it was showing duplicates, the model actually shouldn't be needed. I'll be removing it.